app-accessibility/pipertts/pipertts-1.0.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Piper TTS is an offline text-to-speech program using the NCNN library"
+DESCRIPTION="Piper TTS is an offline text-to-speech program using the NCNN library \
+	to provide natural, Human-like voices. Different voice models can be downloaded from \
+	https://huggingface.co/gyroing/PiperTTS-NCNN-Models/tree/main."
+HOMEPAGE="https://github.com/gyroing/PiperTTS-HaikuOS/"
+COPYRIGHT="2025 GYROING"
+LICENSE="BSD (3-clause)"
+ARCHITECTURES="x86_64"
```

Comment:
Check how this should be organized at: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering

Also take into account new lines (for instance between PROVIDES and REQUIRES).